cross/pip_py3/Makefile

```diff
@@ -0,0 +1,14 @@
+PKG_NAME = pip
```

Comment:
we already have pip for python3/python38, it is cross/pip.
We only have *_py2 cross packages to keep compatible with python2, that hopefully will be removed, when we drop python2 support.